siriuspy/siriuspy/devices/ids.py

```diff
@@ -313,6 +412,22 @@ def kparameter_speed_max_set(self, kparam_speed_max, timeout=None):
         return self._write_sp(
             self.PARAM_PVS.KPARAM_MAXVELO_SP, kparam_speed_max, timeout)
 
+    def kparameter_accel_set(self, kparam_accel, timeout=None):
```

Comment:
I decided to group everything by keywords "kparameter" and "pparameter" prefixes. this is useful for completion. changing these set methods as you suggest will break consistency of the API in this regard.

Reply:
But I still think it's better to change de API, to keep it similar to the APIs from other devices. It is a standard we've been following to have all set methods starting with this word. The same applies for the command methods, which start with cmd, and getter methods, with get.

Reply:
I thing all other devices methods should you the convention I used in this class. This really help me finding methods (which I do not remember) through completion.

Reply:
I don't know, we have a convention we try to follow throughout the code. When righting the library we are not islands. I think the convention we follow is most common among other python libraries we use. See `matplotlib` or `PyQt`, for instance, all their methods start with the action first.

Reply:
@fernandohds564 , I really feel that this refactoring of methods names here is not worthwhile taking people's time. standardization is import but up to a point beyond which the gain / time is too small.

Reply:
I think there is another advantage, either in the completion, if we use the prefixes `set`  and `cmd` as mentioned by @fernandohds564 : easily identify if that name in the completion is a method (that probably requires inputs) or if it is a property. I think this is a standard we would try to keep.

Reply:
@xresende, since time was the only argument left, I tried to do it. I took me 10m35s. I think its ok given the advantages, right? I've made a new branch from this one with the fixes. The name of the branch is `adapt-idff-2-fix-style`. You can merge it into yours if you like.

Reply:
Just for the record, I also agree that we should use the prefixes `set`, `get`, `cmd` in these methods for all the reasons raised by @fernandohds564 and @anacso17. As a `devices` user it would be unnatural to me to search for something like `kparameter_accel_set` if I'm trying to set kparameter. I'd try search for `set` in the completion.

Reply:
> @xresende, since time was the only argument left, I tried to do it. I took me 10m35s. I think its ok given the advantages, right? I've made a new branch from this one with the fixes. The name of the branch is `adapt-idff-2-fix-style`. You can merge it into yours if you like.

I think you misused your precious 10m, @fernandohds564 ! :smile: but since time is irreversible, and will please the majority here, I have nothing left but to go with the flow. thanks for the adaptation!

Reply:
merged your commits in the PR.